core/startos/src/s9pk/v2/pack.rs

```diff
@@ -337,6 +341,22 @@ impl clap::FromArgMatches for ImageConfig {
     }
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize, TS)]
+#[serde(rename_all = "camelCase")]
+#[ts(export)]
+struct EnvVar {
+    env: String,
+}
+
+#[derive(Debug, Clone, Deserialize, Serialize, TS)]
+#[serde(rename_all = "camelCase")]
+#[serde(untagged)]
+#[ts(export)]
+pub enum BuildArg {
+    String(String),
+    EnvVar(EnvVar),
+}
```

Comment:
```suggestion
#[derive(Debug, Clone, Deserialize, Serialize, TS)]
#[serde(rename_all = "camelCase")]
#[serde(untagged)]
#[ts(export)]
pub enum BuildArg {
    String(String),
    EnvVar { 
        env: String,
    }
}
```
I don't think EnvVar warrants its own struct type, given that it is highly contextual

Reply:
much better 👍